cmd/mesh/testdata/manifest-generate/output/pilot_override_values.yaml

```diff
@@ -643,6 +643,478 @@ metadata:
     release: istio
 ---
 
+# Resources for Prometheus component
```

Comment:
This file wasn't modified in https://github.com/istio/operator/pull/286. What PR modified it in `master`?